### DISCUSSION TITLE
LibreOffice / Cryptomator temporary files...

### DISCUSSION THREAD

**Discussion by jotkah** [abraunegg/onedrive#2138] [Q&A] [answered]
Hi Folks,

my LibreOffice files are stored using Cryptomator. When i save a file, a temporary file will be created for some milliseconds. The following things happen:
- the temporary file is created
- the onedrive client sees this file
- the file is deleted
- the onedrive client tried to uload
- i got an error message that the file can't be uploaded

So i got an error message. The file has been deleted in the short time (1-2ms) between your client "sees" this file and decided uploading it.

How can i fix this? Is there an option that upload will only affect on files, which have been untouched for some seconds?

Greetings,
Joerg

**Comment by abraunegg** [comment, marked as answer]
@jotkah 
> How can i fix this? Is there an option that upload will only affect on files, which have been untouched for some seconds?

You need to configure 'skip_file' in your 'config' file to exclude these LibreOffice | Cryptomator temporary files from being attempted to be uploaded.
